File: Python/Fondamentals/Data Types and Variables/gladiator_expenses.py
lost_fights=int(input())
helmet_price=float(input())
sword_price=float(input())
shield_price=float(input())
armor_price=float(input())

total_helmets=lost_fights//2
total_sword=lost_fights//3
total_shield=lost_fights//6
total_armor=total_shield//2
total_price=total_helmets*helmet_price +\
    total_sword*sword_price +\
    total_shield*shield_price +\
    total_armor*armor_price
print(f"Gladiator expenses: {total_price:.2f} aureus")

File: JS Front End/JS Syntax Fundamentals More Exercises/gladiator-expenses.js
function expenses(lostFights, helmetPrice, swordPrice, shieldPrice, armorPrice) {
    let brokenHelmets = 0;
    let brokenSwords = 0;
    let brokenShields = 0;
    let brokenArmors = 0;

    for (let round = 1; round < lostFights + 1; round++) {
        if (round % 2 === 0) {
            brokenHelmets += 1;
        }
        if (round % 3 === 0) {
            brokenSwords += 1;
        }
        if (round % 6 === 0) {
            brokenShields += 1;
        }
    }
    brokenArmors = Math.floor(brokenShields / 2);

    totalPrice = brokenHelmets * helmetPrice +
        brokenSwords * swordPrice +
        brokenShields * shieldPrice +
        brokenArmors * armorPrice

    console.log(`Gladiator expenses: ${totalPrice.toFixed(2)} aureus`)
}

expenses(23,
    12.50,
    21.50,
    40,
    200

)

